src/styles/apply.styl

```diff
@@ -62,4 +62,32 @@
 .team-list {
   padding: 10px;
   margin-bottom: 0;
-}
\ No newline at end of file
+}
+
+// RSVP Section on dashboard
+.rsvp-section {
+  form {
+    margin: 1em 0;
+  }
+  
+  #decline-confirm {
+    padding: 0.5em;
+    margin: 0.5em;
+    height: 0;
+    overflow: hidden;
+    opacity: 0;
+    transition: all 0.3s;
+  }
+  
+  #rsvp-no-first-click {
+    opacity:0;
+  }
+  
+  input[type=checkbox]:checked ~ div#decline-confirm {
+    opacity: 1;
+    height: 7em;
```

Comment:
How do you feel about this height being hard-coded just so we can animate the height of the message container?